speedy-recovery-client/src/components/secured/shared/Header.js

```diff
@@ -78,6 +87,24 @@ class Header extends Component {
       </header>
     );
   }
+
+  componentDidMount() {
+    if (this.props.role !== "Patient") {
+      let id = this.props.userid;
```

Comment:
you should use const instead of let or you can just do getConversation(this.props.userid) since the value is just used once

Reply:
This comment sounds good.